### PR TITLE
[build] Fix signing of published artifacts

### PR DESCRIPTION
**Pull request by Pscheidl**
These artifacts were present in local Nexus after publish:
```
h2o-algos-3.31.0.99999-javadoc.jar | Thu May 14 13:25:27 UTC 2020 | 2244574 |  
-- | -- | -- | --
h2o-algos-3.31.0.99999-javadoc.jar.asc | Thu May 14 13:25:27 UTC 2020 | 650 |  
h2o-algos-3.31.0.99999-javadoc.jar.asc.md5 | Thu May 14 13:25:27 UTC 2020 | 32 |  
h2o-algos-3.31.0.99999-javadoc.jar.asc.sha1 | Thu May 14 13:25:27 UTC 2020 | 40 |  
h2o-algos-3.31.0.99999-javadoc.jar.asc.sha256 | Thu May 14 13:25:27 UTC 2020 | 64 |  
h2o-algos-3.31.0.99999-javadoc.jar.asc.sha512 | Thu May 14 13:25:27 UTC 2020 | 128 |  
h2o-algos-3.31.0.99999-javadoc.jar.md5 | Thu May 14 13:25:27 UTC 2020 | 32 |  
h2o-algos-3.31.0.99999-javadoc.jar.sha1 | Thu May 14 13:25:27 UTC 2020 | 40 |  
h2o-algos-3.31.0.99999-javadoc.jar.sha256 | Thu May 14 13:25:27 UTC 2020 | 64 |  
h2o-algos-3.31.0.99999-javadoc.jar.sha512 | Thu May 14 13:25:27 UTC 2020 | 128 |  
h2o-algos-3.31.0.99999.jar | Thu May 14 13:25:27 UTC 2020 | 1364628 |  
h2o-algos-3.31.0.99999.jar.asc | Thu May 14 13:25:27 UTC 2020 | 650 |  
h2o-algos-3.31.0.99999.jar.asc.md5 | Thu May 14 13:25:27 UTC 2020 | 32 |  
h2o-algos-3.31.0.99999.jar.asc.sha1 | Thu May 14 13:25:27 UTC 2020 | 40 |  
h2o-algos-3.31.0.99999.jar.asc.sha256 | Thu May 14 13:25:27 UTC 2020 | 64 |  
h2o-algos-3.31.0.99999.jar.asc.sha512 | Thu May 14 13:25:27 UTC 2020 | 128 |  
h2o-algos-3.31.0.99999.jar.md5 | Thu May 14 13:25:27 UTC 2020 | 32 |  
h2o-algos-3.31.0.99999.jar.sha1 | Thu May 14 13:25:27 UTC 2020 | 40 |  
h2o-algos-3.31.0.99999.jar.sha256 | Thu May 14 13:25:27 UTC 2020 | 64 |  
h2o-algos-3.31.0.99999.jar.sha512 | Thu May 14 13:25:27 UTC 2020 | 128 |  
h2o-algos-3.31.0.99999.module | Thu May 14 13:25:28 UTC 2020 | 14998 |  
h2o-algos-3.31.0.99999.module.asc | Thu May 14 13:25:28 UTC 2020 | 650 |  
h2o-algos-3.31.0.99999.module.asc.md5 | Thu May 14 13:25:28 UTC 2020 | 32 |  
h2o-algos-3.31.0.99999.module.asc.sha1 | Thu May 14 13:25:28 UTC 2020 | 40 |  
h2o-algos-3.31.0.99999.module.asc.sha256 | Thu May 14 13:25:28 UTC 2020 | 64 |  
h2o-algos-3.31.0.99999.module.asc.sha512 | Thu May 14 13:25:28 UTC 2020 | 128 |  
h2o-algos-3.31.0.99999.module.md5 | Thu May 14 13:25:28 UTC 2020 | 32 |  
h2o-algos-3.31.0.99999.module.sha1 | Thu May 14 13:25:28 UTC 2020 | 40 |  
h2o-algos-3.31.0.99999.module.sha256 | Thu May 14 13:25:28 UTC 2020 | 64 |  
h2o-algos-3.31.0.99999.module.sha512 | Thu May 14 13:25:28 UTC 2020 | 128 |  
h2o-algos-3.31.0.99999.pom | Thu May 14 13:25:27 UTC 2020 | 6632 |  
h2o-algos-3.31.0.99999.pom.asc | Thu May 14 13:25:28 UTC 2020 | 650 |  
h2o-algos-3.31.0.99999.pom.asc.md5 | Thu May 14 13:25:28 UTC 2020 | 32 |  
h2o-algos-3.31.0.99999.pom.asc.sha1 | Thu May 14 13:25:28 UTC 2020 | 40 |  
h2o-algos-3.31.0.99999.pom.asc.sha256 | Thu May 14 13:25:28 UTC 2020 | 64 |  
h2o-algos-3.31.0.99999.pom.asc.sha512 | Thu May 14 13:25:28 UTC 2020 | 128 |  
h2o-algos-3.31.0.99999.pom.md5 | Thu May 14 13:25:27 UTC 2020 | 32 |  
h2o-algos-3.31.0.99999.pom.sha1 | Thu May 14 13:25:27 UTC 2020 | 40 |  
h2o-algos-3.31.0.99999.pom.sha256 | Thu May 14 13:25:27 UTC 2020 | 64 |  
h2o-algos-3.31.0.99999.pom.sha512 | Thu May 14 13:25:27 UTC 2020 | 128 |  
```

### CODE REVIEW

File: gradle/publish.gradle [hunk 1]
```diff
@@ -130,13 +91,17 @@ publishing {
       }
   }
 
+    signing {
```
Comment (honzasterba) [inline gradle/publish.gradle:94]: 👍 
incositent indent

File: gradle/publish.gradle [hunk 2]
```diff
@@ -153,14 +118,14 @@ publishing {
...
+                url "${localNexusLocation}/maven-releases"
               } else {
-                url "${localNexusLocation}/snapshots"
+                url "${localNexusLocation}/maven-snapshots"
               }
 
               credentials {
-                  username project.findProperty("local-release-maven.username") ?: "<NA>"
-                  password project.findProperty("local-release-maven.password") ?: "<NA>"
+                  username "admin"
```
Comment (honzasterba) [inline gradle/publish.gradle:127]: forgotten change

File: gradle/publish.gradle [hunk 3]
```diff
@@ -130,13 +91,17 @@ publishing {
...
+    }
+
   repositories {
       maven {
           name "BuildRepo"
           url "$rootDir/build/repo"
       }
 
-      if (project.ext.isRelease) {
+      if (false) {
```
Comment (honzasterba) [inline gradle/publish.gradle:104]: forgotten

File: gradle/publish.gradle (hunk 2 above)
Comment (Pscheidl) [reply gradle/publish.gradle]: Thanks. Reverted.

File: gradle/publish.gradle (hunk 3 above)
Comment (Pscheidl) [reply gradle/publish.gradle]: Reverted. :pray:

File: gradle/publish.gradle (hunk 1 above)
Comment (Pscheidl) [reply gradle/publish.gradle:94]: I see.

**Pull request merged**